### ISSUE TITLE
přidat adresář k MEDIA_ROOT pro modely

### ISSUE THREAD

**Issue by jachym** [OpenGeoLabs/gismentors-registration#31]
Stav:

Když se ukládá nějaká vec se souborem (Faktura, logo projektu), je uložen bez dalšího do adresáře MEDIA_ROOT

K čemu to vede:

V MEDIA_ROOT se hromadí soubory bez ladu a skladu, které s polu nijak nesouvisí

Co udělat:

Zanořit všechny modely do adresářové struktury odpovídající `MEDIA_ROOT/registration/MODEL_NAME/`. Toto provézt na úrovni modelu, a potom ručně "uklidit" aby to fungovalo.

**Comment by madlenkk** [comment]
Testováíní: založila jsem nový coursetype GeoPython, vypsala kurz, přihlásila se a zkusila vygenerovat certifikáty - to hodilo chybu:
```
Request Method: | POST
-- | --
http://localhost:8000/admin/registration/courseevent/
2.1.5
FileNotFoundError
[Errno 2] No such file or directory: '/tmp/gismentors-certificates-pmclq2f7/registration/coursetype/geopython_logo.png'
/usr/lib/python3.6/shutil.py in copyfile, line 121
/usr/bin/python3
3.6.7
['/home/madlenkk/dev/opengeolabs/gismentors',  '/usr/lib/python36.zip',  '/usr/lib/python3.6',  '/usr/lib/python3.6/lib-dynload',  '/home/madlenkk/.local/lib/python3.6/site-packages',  '/usr/local/lib/python3.6/dist-packages',  '/usr/lib/python3/dist-packages']
pá, 19 Dub 2019 08:51:54 +0000
```

**Comment by madlenkk** [comment]
Testování: 
* nahrála jsem fakturu, uložila se do složky invoices/[UUID] - OK
* nahrála jsem fakturu se stejným názvem, uložila se sufixem - neměla by spíš přepsat tu původní, aby se nám tam nehromadil bordel? - k diskuzi

**Comment by jachym** [comment]
>     * nahrála jsem fakturu se stejným názvem, uložila se sufixem - neměla by spíš přepsat tu původní, aby se nám tam nehromadil bordel? - k diskuzi

No, každá faktura by měla mít vlastní adresář (podle uuid), můžem to brát jako verzování faktur (kdby bylo  potřeba do budoucna) - snad to moc nepořádku v tomhle případě nenadělá?

**Comment by madlenkk** [comment]
Souhlasím, v pohodě.

**Comment by jachym** [comment]
snad opraveno
